.gitignore

```diff
@@ -1,2 +1,3 @@
 build/
 env/
+prjuray/__pycache__
```

Comment:
Just add `__pycache__` and `*.py[co]`?

Reply:
Right. That would be more generic. Fixed.